stacker/commands/stacker/destroy.py

```diff
@@ -0,0 +1,36 @@
+"""Destroys CloudFormation stacks based on the given config.
+
+Stacker will determine the order in which stacks should be destroyed based on
+any manual requirements they specify or output values they rely on from other
+stacks.
+
+"""
+from .base import StackerCommand
+from ...actions import destroy
+
+
+class Destroy(StackerCommand):
+
+    name = "destroy"
+    description = __doc__
+
+    def add_arguments(self, parser):
+        super(Destroy, self).add_arguments(parser)
+        parser.add_argument('-f', '--force', action='store_true',
+                            help="Whehter or not you want to go through "
+                                 " with destroying the stacks")
+
+    def run(self, options, **kwargs):
+        super(Destroy, self).run(options, **kwargs)
+        action = destroy.Action(options.context, provider=options.provider)
+        stack_names = map(lambda x: '  - %s' % (x.fqn,), options.context.get_stacks())
+        message = '\nAre you sure you want to destroy the following stacks:\n%s\n\n(yes/no): ' % (
```

Comment:
Just a convention, but maybe capitalize `No` to indicate that it's the default if you just hit enter.